mock/py/mockbuild/package_manager.py

```diff
@@ -758,5 +758,9 @@ class Dnf5(Dnf):
     """
     name = 'dnf5'
     place_common_opts_after = True
+
+    def execute(self, *args, **kwargs):
+        return super(Dnf, self).execute(*args, **{"pty": False, **kwargs})
```

Comment:
Ok, I'd feel a bit safer if we used:
```
pty = kwargs.pop("pty", False)
```
first, or just (if we want to beat the previous decisions):
```
kwargs["pty"] = False
```

This will prevent us from errors like `got multiple values for keyword argument 'pty'`

Reply:
The `**` should handle that, unless I am missing something.

```
>>> kwargs = {"pty": True}
>>> {"pty": False, **kwargs}
{'pty': True}
```